src/components/Search/Search.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Search.css';
+
+const Search = () => (
+    <div className="searchWrapper">
+        <h1 className="searchTitle">FIND YOUR MOVIE</h1>
+        <div className="search">
+            <input className="searchInput" type="text" placeholder="What do you want to watch?"/>
+            <button className="searchButton">SEARCH</button>
```

Comment:
You are using `text-transform: uppercase;` in filter and sorter, I would recommend using it for other cases as well

Reply:
Will fix.